src/mahoji/commands/runecraft.ts

```diff
@@ -131,6 +133,17 @@ export const runecraftCommand: OSBMahojiCommand = {
 			boosts.push(ringStr);
 		}
 
+		const amuletId = Inventions.find(inv => inv.name === 'Abyssal amulet')!.item.id;
+		if (hasItemsEquippedOrInBank(user, [amuletId])) {
+			const abyssalAmulet = inventionBoosts.abyssalAmulet.boosts.find(b =>
+				b.runes.some(r => stringMatches(r, runeObj.name))
+			);
```

Comment:
this variable would be better named with something describing better what it is, i.e `abyssalAmuletBoost`